addon/components/table-of-contents-plugin/ember-nodes/table-of-contents.hbs

```diff
@@ -1,4 +1,4 @@
-<div class='au-u-background-gray-100 au-u-padding-tiny table-of-contents'>
+<div class='au-u-background-gray-100 au-u-padding-tiny table-of-contents' {{did-insert this.onUpdate}}>
```

Comment:
this makes it so the exported version doesn't update until you've toggled the the ToC on and off (this only runs when that div is inserted)

I assume you did this cause we had the update cycle issues when running it on every state change, which is understandable

and its actually not a bad attempt, thing is we don't really need the exported ToC to be up to date until we actually export...

I think we need to take another look at the ToC component to fix this properly, but I won't block this PR for it cause the rest of the logic is decent

Reply:
yes u are right. I moved the logic to GN, now It's triggered every time before we save the document